alis.conf

```diff
@@ -72,7 +72,7 @@ HOOKS="base !udev !usr !resume !systemd !btrfs keyboard autodetect modconf block
 BOOTLOADER="!grub !refind systemd" # (single)
 
 # desktop
-DESKTOP_ENVIRONMENT="!gnome !kde !xfce !mate !cinnamon !lxde" # (single)
+DESKTOP_ENVIRONMENT="!gnome !kde !xfce !mate !cinnamon !lxde !i3wm" # (single)
```

Comment:
Would it make sense to add `i3-gaps` as well as an option?

Reply:
The [i3](https://www.archlinux.org/groups/x86_64/i3/) package group that I install includes both [i3-wm](https://www.archlinux.org/packages/community/x86_64/i3-wm/) and i3-gaps, and chooses [i3-gaps](https://www.archlinux.org/packages/community/x86_64/i3-gaps/) by default. So this would currently actually install i3-gaps.

It would be a simple enough change to offer both an i3-wm and an i3-gaps option if you think it's a good idea. I'm not sure if it would be adding more options just for the sake of more options? People who specifically want one or the other can also add i3-wm to one of the other package variables like PACKAGES_PACMAN_CUSTOM below.

Reply:
I see that @picodotdev had the same thought. I'll add it as two options